src/terminal-window.c

```diff
@@ -52,6 +52,8 @@ struct _TerminalWindowPrivate
     GtkUIManager *ui_manager;
     guint ui_id;
 
+    GSettings *settings;
```

Comment:
I prefer global_settings instead of settings for the variable name.

Reply:
ok, done